### ISSUE TITLE
Fullscreen bug on Windows

### ISSUE THREAD

**Issue by kuviman** [tomaka/glutin#1071]
Running `--example fullscreen`, I can't alt-tab from the fullscreen window. I think it actually does switch to another window, but the glutin window is still shown on top. Also, pressing `F` to toggle fullscreen works strangely - I still have the fullscreen window shown (but I think the issue is same - the old fullscreen picture is still shown above everything).

The same example from `winit` works fine.

Tested on Windows 7

**Comment by kuviman** [comment]
Looks like there is no problem on Windows 10, so this should be only a Win 7 issue

**Comment by francesca64** [comment]
> Looks like there is no problem on Windows 10, so this should be only a Win 7 issue

That's bad news, since I don't have a Windows 7 installation to test with. How about you, @Osspial?

**Comment by Osspial** [comment]
I do not, unfortunately. Setting up a Windows 7 VM shouldn't be too difficult though, so I can see if I can reproduce it in that.

**Comment by kuviman** [comment]
Hm, actually, I noticed a couple things.. First, there should be a triangle shown in this example, but there isn't.

So, if I change the example to start in non-fullscreen, I see the triangle, but once I switch to fullscreen it's gone.

Alt-tabbing does work, but it only works for some apps (like `cmd.exe` for example, or switching to desktop), and does not work for others.

Basically, when I go fullscreen, some apps stop updating visually, including the app itself (no more triangle), browsers (both chrome and firefox) and VS Code.

I don't know if browsers or code (which is electron-based, so also browser I guess) uses opengl or not, but I think what happens is that all opengl stops working (for example if I open another instance of an example with a triangle, it also stops working).

Interestingly, when I close the app that was is fullscreen, other apps (including another instance of the example) start working again.

Here's what it looks like:
![image](https://user-images.githubusercontent.com/5872782/46963630-37e2c080-d0b7-11e8-8540-da7baa8a18ce.png)

**Comment by Osspial** [comment]
@kuviman It looks like you have Aero disabled in that screenshot. Does the bug happen with Aero enabled?

**Comment by kuviman** [comment]
Oh, Aero was probably disabled because I tried connecting remotely to test. If I use the PC directly, it is like what I described initially: when I switch from windowed to fullscreen, I see the triangle but the image stays on top whatever I do (I don't see the switching widget on alt-tab, taskbar or anything until I close the app).

I tried running on another Win7 PC, no issues, only on this one. I tried using gputest from geeks3d and it has similar issue. Although some random UE4 game doesn't have it (but it uses directx instead).

So, I guess this is not glutin's fault and this should be closed.
